src/library/simplified/extension/doubleclick-src-cat-type-PHIS.js

```diff
@@ -0,0 +1,306 @@
+if(b.siteId && utag.isPIS_FH() ){
```

Comment:
@v-roroy does isPIS() also covers FHC? I looked at isPIS() and it doesn't cover FHC.